src/Microsoft.Health.Dicom.Core/Registration/DicomServerApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,66 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Configs;
+using Microsoft.Health.Dicom.Core.Modules;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.IO;
+
+namespace Microsoft.Health.Dicom.Core.Registration;
+
+public static class DicomServerApplicationBuilderExtensions
+{
+    private const string DicomServerConfigurationSectionName = "DicomServer";
+
+    public static IDicomServerBuilder AddDicomServer(this IServiceCollection services,
```

Comment:
We could potentially rename this as something like `AddDicomCore` too if we wanted.

Reply:
That makes sense to me.